core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/jdk/JdkQueries.java

```diff
@@ -153,8 +153,8 @@ public final class JdkQueries {
 			.select(ALLOC_INSIDE_TLAB_AVG).select(ALLOC_INSIDE_TLAB_SUM).groupBy(ALLOCATION_CLASS).build();
 	public static final IItemQuery ALLOC_INSIDE_TLAB_BY_THREAD = fromWhere(JdkFilters.ALLOC_INSIDE_TLAB)
 			.select(ALLOC_INSIDE_TLAB_AVG).select(ALLOC_INSIDE_TLAB_SUM).groupBy(EVENT_THREAD).build();
-	public static final IItemQuery JFR_DATA_LOST = fromWhere(JdkFilters.JFR_DATA_LOST)
-			.select(END_TIME, EVENT_THREAD, FLR_DATA_LOST).build();
```

Comment:
Removed the unnecessary usage of Event Thread here which was causing issue. Since there was no event thread attribute for jdk.DataLoss event, hence there was no matching accessor also and it was throwing NPE due to null accessor when report was generated using verbose option. Issue was present for both XML and HTML report generation but customer has reported it for XML only.

Reply:
When dumping the recording, there is no event thread in the metadata of the event, so removing EVENT_THREAD sounds right, but shouldn't you leave the query?

Reply:
Ah. Missed the +part below. All good. :)